tests/test_update_max_potential_damage_user_data.py

```diff
@@ -0,0 +1,73 @@
+from hydromt_fiat.fiat import FiatModel
+from hydromt.log import setuplog
+from pathlib import Path
+import pytest
+import shutil
+import copy
+import numpy as np
+
+
+EXAMPLEDIR = Path().absolute() / "examples" / "data" / "update_max_potential_damage"
+DATADIR = Path().absolute() / "hydromt_fiat" / "data"
+
+_cases = {
+    "update_max_potential_damage_with_points": {
+        "dir": "fiat_model",
+        "new_root": EXAMPLEDIR / "test_update_max_potential_damage_points",
+        "data_catalog": DATADIR / "hydromt_fiat_catalog_USA.yml",
+        "max_potential_damage_file": EXAMPLEDIR
```

Comment:
You're using the same geometry files for this test as for the update_ground_floor_height test. Is that the idea? Maybe it does not matter for the testing itself but the naming is confusing.

Reply:
I modified the files names to make them clearer